tests/integration/test_processes.py

```diff
@@ -13,6 +14,14 @@
 from tests.utils import filter_relevant_pymalloc_allocations
 
 
+@pytest.fixture(scope="module", autouse=True)
+def set_multiprocessing_to_fork():
+    current_method = multiprocessing.get_start_method()
+    multiprocessing.set_start_method("fork", force=True)
+    yield
+    multiprocessing.set_start_method(current_method, force=True)
```

Comment:
`force=True` is undocumented - any risk to this breaking, or going away in the future?

Reply:
> force=True is undocumented - any risk to this breaking, or going away in the future?

Very low risk. If it breaks, we will set it globally in the test suite. This is just a cleaner way.